src/lib/stark/ledger/createSignedTransfer.js

```diff
@@ -31,6 +31,7 @@ module.exports = async (
   if (transferTokenAddress) {
     const tokenInfo = byContractAddress(transferTokenAddress)
     transferTokenAddress = transferTokenAddress.substr(2)
+    console.log('tokenInfo ', tokenInfo)
```

Comment:
guess we dont need these console.logs

Reply:
@plutoegg I am keeping some console logging for this change as this is for debugging. console logging will be removed when we have tested.